report/22_Process-Team-Organization.md

```diff
@@ -1 +1,5 @@
-How is the team organized?
+The team has organized itself mainly through a weekly meeting after the lecture in which tasks were distributed. The team has not found any formalized process corresponding to this, but it has had several advantages and drawbacks.
+
+On the positive side, this method has very little overhead, and requires little scheduling since the main activity is done in continuation of a mandatory activity for which members are supposed to be present. It also leaves more time for actual development work. 
+
+The method does, however, suffer a few problems. It does not involve any dedicated time for working, which makes it harder to schedule activities like pair programming, which has been done a few times throughout the course. There is no mechanic to ensure that more overreaching goals are reached, which by coincidence was not a major problem in this course. This is likely due to the fact that the team managed to get through quite a few tasks in the first few weeks, and could thus keep up with the scheduled tasks.
```

Comment:
When did we pair program?
I would argue that there was a mechanic to ensure that the "overreaching goals" (assigned tasks from teachers right) were reached. We were given them at the end of the lecture and made a plan for how to get them done by next week. Also people were haunted in the Discord and until they did their assigned tasks.

Reply:
I have paired with Tobias at least two times. I don't know if anybody else have done it.